esp32-cam-mjpeg-multiclient.ino

```diff
@@ -432,6 +431,8 @@ void setup()
     ESP.restart();
   }
 
+  // disable brownout detector
+  WRITE_PERI_REG(RTC_CNTL_BROWN_OUT_REG, 0);
```

Comment:
Some chips dont handle wifi power consumption and trigger a brownout restart; this disables that till after wifi is started